package.json

```diff
@@ -46,6 +46,9 @@
     "url": "https://github.com/pioardi/poolifier/issues"
   },
   "homepage": "https://github.com/pioardi/poolifier#readme",
+  "files": [
+    "lib"
```

Comment:
are we sure that this is going to work ? I mean should be lib/* or something like ?

Reply:
Yes, you can check it via `npm publish --dry-run`